src/main/java/uk/gov/gchq/gaffer/doc/operation/FilterExample.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.doc.operation;
+
+import uk.gov.gchq.gaffer.data.element.Element;
+import uk.gov.gchq.gaffer.data.element.function.ElementFilter;
+import uk.gov.gchq.gaffer.operation.OperationChain;
+import uk.gov.gchq.gaffer.operation.OperationException;
+import uk.gov.gchq.gaffer.operation.impl.function.Filter;
+import uk.gov.gchq.gaffer.operation.impl.get.GetAllElements;
+import uk.gov.gchq.koryphe.impl.predicate.IsLessThan;
+import uk.gov.gchq.koryphe.impl.predicate.IsMoreThan;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FilterExample extends OperationExample {
+    public static void main(final String[] args) throws OperationException {
+        new FilterExample().run();
+    }
+
+    public FilterExample() {
+        super(Filter.class);
+    }
+
+    @Override
+    public void runExamples() {
+        filterEdgesByCount();
```

Comment:
These examples demonstrate the way to configure a Filter operation. However, it should never really be used after a GetAllElements operation as it is far more efficient to add the filters directly to the GetAllElements operation. 

It is more intended to be used after operations that don't have a View so filters can't be applied directly. For example:

```java
new OperationChain.Builder()
   .first(new NamedOperation.Builder<EntityId, CloseableIterable<EntityId>>()
                .name("2-hop")
                .input(new EntitySeed(1))
                .build())
   .then(new Filter.Builder()
         .globalEdges(new ElementFilter.Builder()
                      .select("count")
                      .execute(new IsMoreThan(2))
                      .build())
         .build())
   .build();
```

or 

```json
final OperationChain<Iterable<?>> opChain = new OperationChain.Builder()
    .first(new GetJobResults.Builder()
        .jobId(jobId)
        .build())
    .then(new Filter.Builder()
         .globalEdges(new ElementFilter.Builder()
                      .select("count")
                      .execute(new IsMoreThan(2))
                      .build())
         .build())
    .build();
```